src/js/components/Diagram/doc.js

```diff
@@ -54,6 +76,18 @@ export const doc = Diagram => {
 };
 
 export const themeDoc = {
+  'diagram.animation': {
+    description: 'Any animation specific configuration for Diagram.',
```

Comment:
"Any animation" is not so accurate, It's only applicable for 'draw' and 'pulse'.

Reply:
Updated the text for this theme prop, please let me know what you think